lib/game.rb

```diff
@@ -0,0 +1,64 @@
+require 'csv'
+require './lib/games'
+
+class Game
+  attr_reader :game_id, :season, :type, :date_time, :away_team_id, :home_team_id, :away_goals, :home_goals, :venue, :venue_link
```

Comment:
Great start. I think some of these may not be needed as far as being read but we can keep it until we are done building everything to see what we don't need.

Reply:
This looks good!